README.md

```diff
@@ -13,7 +13,7 @@ The SDK supports PHP 5.6 or major
 #### Using Composer
 
 1. Download [Composer](https://getcomposer.org/download/) if not already installed
-2. Go to your project directory and run `composer require "mercadopago/dx-php:1.2.1"` on the command line.
```

Comment:
We specify the version to allow to users identify easily which version are installing to get appropriate support when they need, I suggest add the `--prefer-dist` as a pro tip below of the steps.

Reply:
Ok @joelibaceta